src/v2/view-builder/internals/BaseView.js

```diff
@@ -43,6 +43,7 @@ export default View.extend({
     const IonModel = this.createModelClass(
       this.options.currentViewState,
       optionUiSchemaConfig);
+
     const model = new IonModel ({
```

Comment:
How about change this to 
```this.model = new IonModel ({```
And remove line #55, then change the rest of occurrences of `model` to `this.model` ?

Reply:
There are many places need to update to use `this.model` hence I'm being a little lazy to just make minimal changes. :D